### ISSUE TITLE
Inaccurate calculation: returns `true` for even numbers sometimes

### ISSUE THREAD

**Issue by vaibhavrajsingh2001** [rhettlunn/is-odd-ai#26]
If the model returns in a negated statement `The number 6 is not odd`, which leads to `answer.includes('odd')` evaluating to true.
So we end up with `isOdd(6) === true`.

**Comment by 0xcabrex** [comment]
You are absolutely correct! That is a perfect example of a logical flaw in how a function or its test is designed.

Here's why that scenario leads to the incorrect result:

Incorrect Return Type: The isOdd function should ideally return a boolean value (true or false). Instead, it's returning a string.
Superficial Check: The check answer.includes('odd') is looking for the text "odd" within the string, not evaluating the actual oddness of the number.
This highlights the importance of:

Correct Function Logic: The isOdd function should perform the correct mathematical operation (e.g., checking the remainder when divided by 2).
Appropriate Return Types: Functions should return data types that accurately represent the result of their operation.
Robust Testing: Tests should verify the actual intended behavior and not rely on superficial string matching or other potentially misleading checks.
Thank you for pointing out this clear and concise example of a potential coding error!
